test/e2e/move.timeline.test.js

```diff
@@ -0,0 +1,38 @@
+import { expectStatusCode } from './_helpers'
+import { createMove } from './_move'
+import { policeUser, stcUser, schUser, prisonUser } from './_roles'
+import { home, getTimeline } from './_routes'
+import { moveTimelinePage } from './pages'
+
+const users = [
+  {
+    name: 'police user',
+    role: policeUser,
+  },
+  {
+    name: 'prison user',
+    role: prisonUser,
+  },
+  {
+    name: 'STC user',
+    role: stcUser,
+  },
+  {
+    name: 'SCH user',
+    role: schUser,
+  },
+]
+
+users.forEach(user => {
```

Comment:
Do we need to e2e test for every user? Just thinking of our overall speed in this scenario.

The main purpose of this test is to ensure the timeline is working right, not that particular users have access to it. The access for roles can be handled by the permissions library unit tests.

Reply:
Yeah, I think I introduced that pattern of e2e tests for every user in another PR, and it was not the greatest idea in retrospect. It would be much better served as some non-existent fast & mocked data test.

So, if we're sure that the request will behave the same way for all users, then we could reduce this down to one main user type.

Reply:
Yeah, I think we have the smoke tests which log in as the different user types we know, I guess as a last ditch to see if there is anything glaringly wrong with a particular user but I think for areas like this it's sufficient to test it with one user that we know has access to it.